### PR TITLE
[WIP] Implement org and section lookup (including url rules)

### LINKED ISSUE #18: URL lookup on gov.uk content api
Process should look like:
* Raw.full_url is cleaned with the clean_url() function based on a set of rules.
* Cleaned urls are looked up using the content API
* Duplicates (based on full_url, sections, orgs, are dropped)

A date of lookup is included to be used for linking back to the full_url with time relevance.

### PR DESCRIPTION
**Pull request by ivyleavedtoadflax**
Implement functionality to close #18.

### CODE REVIEW

**Comment by ivyleavedtoadflax** [comment]
Method for bulk lookup of Urls:

```
# Extract all full_url from raw entries

raw = Raw.query.all()
urls = [i.full_url for i in raw]

# Remove duplicates by list->set->list

unique_urls = list(set(urls))

# Apply url cleaning rules (and create Urls objects)

clean_urls = [clean_url(i, Urls) for i in unique_urls if i is not None]

# Perform API lookup and integrate results into objects

urls_with_org = [get_org(i for i in clean_urls)]

# Add new objects to the database
db.session.bulk_save_objects(urls_with_org)
db.commit()

# Now drop any duplicates that have been added to the table

db.engine.execute('delete from urls u where exists (select 1 from urls u1 where u.full_url = u1.full_url and u.org0=u1.org0 and u.section0 = u1.section0);')
db.session.commit()

```

File: README.md [hunk 1]
```diff
@@ -38,7 +38,11 @@ See [heroku docs](https://devcenter.heroku.com/articles/connecting-to-heroku-pos
 
 * Set up a heroku pipeline to detect pushes on master to github
 * Push to github, heroku will detect, and build the app
+<<<<<<< cd60c9cd167f49c44fa88848407b6cd98cb80d66
```
Comment (davidread) [inline README.md]: looks like a bad merge here

**Comment by ivyleavedtoadflax** [comment]
Yeah, this or PR needs work and a rebase!

**Comment by ivyleavedtoadflax** [comment]
Closing this for now.

**Pull request closed without merging**